.prettierignore

```diff
@@ -1 +1,2 @@
 .next
+*.yaml
```

Comment:
why not format yaml files anyway?

Reply:
Good question, bad assumption on my part. Thought prettier didn't support YAML